frontend/src/main-components/CreateCardContent.js

```diff
@@ -61,7 +65,7 @@ class CreateCardContent extends Component {
           if (prevState.toLang != toLang || prevState.text != text) {
             return;
           }
```

Comment:
any reason not to setState loading=false right after getTranslation returns?

Reply:
This means that the user typed in a different word and the request is still being processed.

Reply:
Add comment?

Reply:
There is a comment on line 60 but it's cut off on the preview for some reason.